README.md

```diff
@@ -90,23 +90,30 @@ There are entry point scripts for each of the three app types (`basic.ts`, `fit.
 built separately. You can build an individual app type using e.g. `npm run build-basic`. Each entry script invokes a store
 and top-level component for the given app type. 
 
+### Unit Tests
 Run unit tests from `app/static` using `npm run test:unit`. Run [eslint](https://eslint.org/) with `npm run lint` or `npm run lint:fix`.
 
-### Back end
+## Backend
 
 The back end handles requests for apps by reading the corresponding config files and rendering an html [Handlebars](https://handlebarsjs.com/) template
 `views/app.hbs` to use the appropriate js file, and also to provide configuration values to the relevant store via
 the `appConfig` object.
 
 The back-end server can be built using `npm run build` and run using `npm run serve`, both from `app/server`.
 
+### Unit Tests
 Run tests from `app/server` using `npm test`. Run [eslint](https://eslint.org/) with `npm run lint` or `npm run lint:fix`.
 
-### Browser tests
+
+### Integration Tests
+
+From `app/server` using `npm run integration-test`
+
+## Browser tests
 
 Browser tests use [Playwright](https://playwright.dev/). Run browser tests, while the app is running, using `npm run e2e-test` from `app/static`.
 
-### Publishing
+## Publishing
```

Comment:
This change now makes it look like 'Usage via docker' and 'Quickly test drive a feature branch' are sub-sections of 'Publishing' which isn't the case.